### PR TITLE
add POSTGRES_PASSWORD to docker-compose.yml and in contributing docum…

### PR DESCRIPTION
**Pull request by charlesaracil-ulti**
…entation

Cf. #176 

### Successful PR Checklist:
- [ ] Tests
- [ ] Documentation (optionally: [run spell checking](https://github.com/peopledoc/procrastinate/blob/master/CONTRIBUTING.rst#build-the-documentation))
- [ ] Had a good time contributing? (if not, feel free to give some feedback)

### CODE REVIEW

**Comment by elemoine** [comment]
Bonus: the change is backward compatible, in the sense that `docker-compose up postgres` will work even without pulling the new `postgres:10` image.

File: docker-compose.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ services:
       PGDATABASE: procrastinate
       PGHOST: postgres
       PGUSER: postgres
+      POSTGRES_PASSWORD: password
```
Comment (elemoine) [inline docker-compose.yml]: This should actually be set at postgres service level.

Comment (charlesaracil-ulti) [reply docker-compose.yml]: you mean alongside the variable `POSTGRES_DB` ? It's not a environment variable ?

Comment (elemoine) [reply docker-compose.yml]: You need `PGPASSWORD` in the procrastinate env vars (in the same way we set `PGPASSWORD` for `psql`), and `POSTGRES_PASSWORD` in the postgres env vars.

Comment (charlesaracil-ulti) [reply docker-compose.yml]: it should be done thank you :)

**Comment by elemoine** [comment]
Thanks @charlesaracil-ulti!

**Comment by charlesaracil-ulti** [comment]
Thanks for you reviews and explanations !

**Pull request merged**